packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -131,6 +134,14 @@ export namespace WorkspaceCommands {
         category: 'File',
         label: 'Save As...',
     };
+    export const COPY_PATH: Command = {
+        id: 'core.copy.path',
+        label: 'Copy Path'
+    };
+    export const COPY_RELATIVE_PATH: Command = {
```

Comment:
I believe we should also provide keybindings for these new commands (`COPY_PATH` and `COPY_RELATIVE_PATH`), similarly to VSCode.